colibre/scripts/SNII_density_distribution.py

```diff
@@ -0,0 +1,127 @@
+"""
+Plots the SNII density distribution.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import unyt
+
+from unyt import mh, cm
+
+from swiftsimio import load
+
+try:
+    plt.style.use("mnras.mplstyle")
+except:
+    pass
+
+from swiftpipeline.argumentparser import ScriptArgumentParser
+
+arguments = ScriptArgumentParser(
+    description="Creates a plot showing the distribution of the gas densities recorded when the gas was last heated by SNII, split by redshift"
```

Comment:
This is quite long, can you split it up as:

```suggestion
    description=(
        "Creates a plot showing the distribution of the gas densities "
        "recorded when the gas was last heated by SNII, split by redshift"
    )
```

Reply:
Should be done now. There are also a few other places in the code where the limit of 88 characters per line is exceeded. I can change these as well if needed.

Reply:
the rest are fine if they are just strings, this one was just particularly egregious.